packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/openidconnect/test-instance.md

```diff
@@ -4,11 +4,11 @@
 4. Click the **Sign On** tab to view and copy the OIDC client ID and secret.
 5. Click **View Setup Instructions** to open a new tab to your integration setup instructions. This is the customer configuration guide that you previously specified in the OIN Wizard.
 6. Follow the instructions in your guide to set up the SSO integration on your app with the OIDC client ID and secret provided.
-7. Follow these steps if you have an Identity Engine Developer Edition org:
+7. Follow these steps if you have an Identity Engine Okta Integrator Free Plan org:
    1. Click the **Sign On** tab, scroll to the **User authentication** section, and click **Edit**.
    1. Select **Password only** from the **Authentication policy** dropdown menu.
    [[style="list-style-type:lower-alpha"]]
    1. Click **Save**.
-   > **Note:** Most recent Okta Developer Edition orgs are Identity Engine orgs. See [OIN Wizard authentication policy for testing](/docs/guides/submit-app-prereq/main/#oin-wizard-authentication-policy-for-testing).
+   > **Note:** Most recent Integrator Free Plan orgs are Identity Engine orgs. See [OIN Wizard authentication policy for testing](/docs/guides/submit-app-prereq/main/#oin-wizard-authentication-policy-for-testing).
```

Comment:
Aren't they all OIE orgs now? If so - remove "Most recent"

Reply:
We almost don't need the note, but remove the "most recent"